### ISSUE TITLE
weight decay

### ISSUE THREAD

**Issue by h-amirkhani** [rcmalli/keras-vggface#58]
Is there any way to add weight decay to the implemented vggface? Since a weight decay of 0.0005 is reported in the [original paper](https://www.robots.ox.ac.uk/~vgg/publications/2015/Parkhi15/parkhi15.pdf).
